### PR TITLE
Add support Laravel 6.0

### PR DESCRIPTION
**Pull request by khanhvu14**
Add support Laravel 6.0

### CODE REVIEW

**Comment by vipertecpro** [comment]
Congratulations on more simplicity. <3 Really happy today.

File: composer.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
   ],
   "require": {
     "php": "^5.6 || ^7.0",
-    "illuminate/support": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.*"
+    "illuminate/support": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || 6.0.*"
```
Comment (joaorobertopb) [inline composer.json]: **6.0.*** is the equivalent of **>=6.0 <6.1** !

See: https://getcomposer.org/doc/articles/versions.md#wildcard-version-range-

Comment (joaorobertopb) [reply composer.json]: Hey @renatomarinho, I suggest replace **6.0.*** for **ˆ6.0**

**Comment by cihantas** [comment]
Can we tag this, so it's available in composer?

**Comment by joaorobertopb** [comment]
@cihantas released version 1.8.11

**Pull request merged**